composer.json

```diff
@@ -39,7 +39,8 @@
     "require-dev": {
         "fzaninotto/faker": "^1.5",
         "graham-campbell/testbench-core": "^1.0",
-        "phpunit/phpunit": "^4.7.6"
+        "phpunit/phpunit": "^4.7.6",
+        "laravel/homestead": "dev-master"
```

Comment:
I understand that is still an acceptable way to use it, but you can't really distribute anything with that setup. 

If you are very familiar with Laravel and the environment, sure you can set up the global homestead instance and add Cachet and reprovision, etc. Personally I would rather have everything necessary in the project so someone can just run `vagrant up` and not need worry about changing things outside of the project.

If this isn't something you are interested in, I understand. Go ahead and decline the PR and I will just continue using the global installation.